### PR TITLE
Add sig-docs-ta members

### PR DESCRIPTION
**Pull request by kittenking**
Add owner and reviewer for the Tamil team.

### CODE REVIEW

**Comment by nikhita** [comment]
@kittenking thanks!

/assign @zacharysarah 

/hold
until https://github.com/kubernetes/website/pull/15485 is merged

File: config/kubernetes/sig-docs/teams.yaml [hunk 1]
```diff
@@ -173,14 +185,15 @@ teams:
...
-    - mittalyashu #hindi
+    - mittalyashu # Hindi
     - nasa9084 # Japanese
     - raelga # Spanish
     - remyleone # French
     - rlenferink # Italian
     - tnir # Japanese
     - zacharysarah # English
     - zhangxiaoyu-zidif # Chinese
+    - kittenking # Tamil
```
Comment (cblecker) [inline config/kubernetes/sig-docs/teams.yaml]: `config_test.go:250: The team sig-docs-l10n-admins in org Kubernetes has an unsorted list of members`

Comment (kittenking) [reply config/kubernetes/sig-docs/teams.yaml]: @cblecker Sorted now!

**Comment by nikhita** [comment]
@kittenking can you squash the commits into a single commit? lgtm otherwise :+1:

**Pull request closed without merging**